frontend/package.json

```diff
@@ -9,15 +9,21 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@emotion/cache": "^11.11.0",
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.0",
```

Comment:
What is `@emotion` package for?

Reply:
Emotion is a JavaScript library that allows you to write CSS styles directly in your JavaScript code. This approach offers benefits like easy style composition and a smoother developer experience. 
 i have used  "@emotion/cache": "^11.11.0",  becusae 
In simple terms, @emotion/cache is like a memory bank for styles in Emotion. It stores styles that have been used before, so when your website or app loads, it doesn't have to recreate those styles from scratch every time. This makes your website faster because it doesn't have to do unnecessary work, and it helps prevent duplicate styles from cluttering up your code. Overall, it's a way to make your styling process more efficient and your website faster.
 
"@emotion/react": "^11.11.4",
In simple terms, @emotion/react is a package that allows you to style your React components in a more organized and efficient way. It provides tools that make it easier to write and manage CSS styles directly within your React code. This means you can make your components look exactly how you want them to without needing separate CSS files.

"@emotion/styled": "^11.11.0",
@emotion/styled is a package that lets you create styled components in your React applications using Emotion. It's like a special tool that makes it easy to style your components by writing CSS inside your JavaScript or TypeScript files. This way, you can keep your styling code close to your component code, making it easier to manage and understand.

Reply:
below are the codes one with using @emotion lib and one without 

/** With @emotion/styled **/

import styled from '@emotion/styled';

const Button = styled.button`
  background-color: #007bff;
  color: #ffffff;
  padding: 8px 16px;
  border: none;
  border-radius: 4px;
  cursor: pointer;

  &:hover {
    background-color: #0056b3;
  }
`;

const App = () => {
  return (
    <div>
      <Button>Click me</Button>
    </div>
  );
};

export default App;

without emotion 

/** Without @emotion/styled **/

const buttonStyles = {
  backgroundColor: '#007bff',
  color: '#ffffff',
  padding: '8px 16px',
  border: 'none',
  borderRadius: '4px',
  cursor: 'pointer',
};

const hoverStyles = {
  backgroundColor: '#0056b3',
};

const App = () => {
  return (
    <div>
      <button style={buttonStyles}>Click me</button>
    </div>
  );
};

Reply:
Can we remove it?